Libraries/Components/ViewPager/ViewPagerAndroid.android.js

```diff
@@ -72,104 +70,93 @@ export type ViewPagerScrollState = $Enum<{
  * }
  * ```
  */
-class ViewPagerAndroid extends React.Component<{
-  initialPage?: number,
-  onPageScroll?: Function,
-  onPageScrollStateChanged?: Function,
-  onPageSelected?: Function,
-  pageMargin?: number,
-  peekEnabled?: boolean,
-  keyboardDismissMode?: 'none' | 'on-drag',
-  scrollEnabled?: boolean,
-}> {
-  /* $FlowFixMe(>=0.78.0 site=react_native_android_fb) This issue was found
-   * when making Flow check .android.js files. */
-  static propTypes = {
-    ...DeprecatedViewPropTypes,
-    /**
-     * Index of initial page that should be selected. Use `setPage` method to
-     * update the page, and `onPageSelected` to monitor page changes
-     */
-    initialPage: PropTypes.number,
 
-    /**
-     * Executed when transitioning between pages (ether because of animation for
-     * the requested page change or when user is swiping/dragging between pages)
-     * The `event.nativeEvent` object for this callback will carry following data:
-     *  - position - index of first page from the left that is currently visible
-     *  - offset - value from range [0,1) describing stage between page transitions.
-     *    Value x means that (1 - x) fraction of the page at "position" index is
-     *    visible, and x fraction of the next page is visible.
-     */
-    onPageScroll: PropTypes.func,
+type Props = $ReadOnly<{|
+  /**
```

Comment:
The propTypes version spreads in `DeprecatedViewPropTypes` so we probably need to include the view props here too.

```
import type {ViewProps} from 'ViewPropTypes';

type Props = $ReadOnly<{|
  ...ViewProps,
  // ...
|};
```

Reply:
Thanks, thanks to the feedback I'll be editing